### DISCUSSION TITLE
Unified Alerting Telegram Images

### DISCUSSION THREAD

**Discussion by exander77** [grafana/grafana#54282] [Alerting]
Where can We expect the ability to use Telegram with Unified Alerting?
For us, alerts without images are pretty much useless.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
